### ISSUE TITLE
Move to aria-roledescription once lighthouse is okay with it

### ISSUE THREAD

**Issue by alexreardon** [atlassian/react-beautiful-dnd#1742]
We should be good to go add `aria-roledescription` after this upgrade of axe releases in lighthouse: https://github.com/GoogleChrome/lighthouse/pull/10056

**Comment by RavenAlly** [comment]
Lighthouse 6.0.0 (shipped with Chrome 84) contains axe-core 3.5.3.
I was able to run Lighthouse and maintain a 100 a11y score on a draggable item with `role` and `aria-roledescription` defined.
